packages/components/src/test-utils/addParentAndSetRequiredProps.ts

```diff
@@ -22,27 +21,35 @@ export const addParentAndSetRequiredProps = (tagName: TagName, component: any):
     component[childTagName] = child;
   }
 
-  if (requiredNamedSlots) {
-    requiredNamedSlots.forEach(({ slotName, tagName: requiredTagName }) => {
-      const child = document.createElement(requiredTagName);
+  const requiredNamedSlots = Object.entries(slotsMeta ?? {}).filter(([, value]) => value.isRequired);
```

Comment:
A little bit easier to read and to avoid calling filter on an empty array would be would be 
```js
const requiredNamedSlots = slotsMeta ?? Object.entries(slotsMeta).filter(([, value]) => value.isRequired);
```

and below 

```js
if (requiredNamedSlots && requiredNamedSlots.length > 0) {
```